packages/code-du-travail-frontend/src/outils/utils.js

```diff
@@ -0,0 +1,72 @@
+const createValuesMatcher = values => item => {
```

Comment:
I would prefer named the file situation.{utils|lib}.js or just situation.js  and remane local situations.js to  another name (it only contains mapping between string / constants) for display purpose

Reply:
OK i understand and i'm agree with you

Reply:
I think it's more clear to keep situations.js on local and rename utils.js to situations.utils.js

Reply:
ok with that but all the situation logic should go into situation.utils

Reply:
Yes

Reply:
Done